android/app/src/debug/java/no/mittatb/ReactNativeFlipper.java

```diff
@@ -26,13 +26,16 @@
 import okhttp3.OkHttpClient;
 import tech.bam.rnperformance.flipper.RNPerfMonitorPlugin;
 
+/**
+ * Class responsible of loading Flipper inside your React Native application. This is the debug
+ * flavor of it. Here you can add your own plugins and customize the Flipper setup.
+ */
 public class ReactNativeFlipper {
   public static void initializeFlipper(Context context, ReactInstanceManager reactInstanceManager) {
     if (FlipperUtils.shouldEnableFlipper(context)) {
       final FlipperClient client = AndroidFlipperClient.getInstance(context);
 
       client.addPlugin(new InspectorFlipperPlugin(context, DescriptorMapping.withDefaults()));
-      client.addPlugin(new ReactFlipperPlugin());
```

Comment:
This one was added by @gorandalum if I am not wrong.

Reply:
I think this is moved to be handled by the flipper client base, so it's not necessary to include anymore 🤔